mods/ra2/rules/allied-infantry.yaml

```diff
@@ -125,26 +125,64 @@ e1:
 		HP: 125
 	Mobile:
 		Speed: 31
+		RequiresCondition: !chronodisable && undeployed
 	Passenger:
 		PipType: Green
 	RevealsShroud:
 		Range: 5c0
-	AttackFrontal:
-		Voice: Attack
 	Armament@primary:
 		Weapon: M60
-		RequiresCondition: !rank-elite
+		RequiresCondition: !rank-elite && !deployed
 	Armament@elite:
 		Weapon: M60E
-		RequiresCondition: rank-elite
-	# Armament@deployed:
-	#	Weapon: para
-	#	UpgradeTypes: deployed
-	#	UpgradeMinEnabledLevel: 1
-	# Armament@elite-deployed:
-	#	Weapon: paraE
-	#	UpgradeTypes: eliteparaweapon
-	#	UpgradeMinEnabledLevel: 1
+		RequiresCondition: rank-elite && !deployed
+	Armament@deployed:
+		Weapon: para
+		Name: deployed
+		RequiresCondition: !rank-elite && deployed
+	Armament@elite-deployed:
+		Weapon: paraE
+		Name: deployed
+		RequiresCondition: rank-elite && deployed
+	GrantConditionOnDeploy:
+		DeployedCondition: deployed
+		UndeployedCondition: undeployed
+		CanDeployOnRamps: True
+		Facing: 160
+		RequiresCondition: !parachute
```

Comment:
We do want to be able to give the deploy command while they are paradropping (when they hit the ground they should immediately deploy if given the command in the air).

Is this not possible technically yet?

Reply:
It isn't possible yet. If i leave this out, it would make them deploy in air.

Reply:
Let's create an upstream ticket and link it too, then.